File: src/test/resources/py/MissingImports.py
class MissingImports:
	def print_random_number(self):
		print(random.randint(0, 100))

	def print_current_date(self):
		print(datetime.now().strftime("%Y-%m-%d"))

	def print_file_exists(self):
		print(os.path.exists("example.txt"))

	def print_local_date(self):
		print(date.today())

	def print_zone_id(self):
		print(time.tzname)

File: src/test/resources/go/MissingImports.go
package main

type MissingImports struct{}

func (self MissingImports) printRandomNumber() {
	fmt.Println(rand.Intn(100))
}

func (self MissingImports) printCurrentDate() {
	fmt.Println(time.Now().Format("2006-01-02"))
}

func (self MissingImports) printFileExists() {
	_, err := os.Stat("example.txt")
	if err == nil {
		fmt.Println(true)
	} else if os.IsNotExist(err) {
		fmt.Println(false)
	} else {
		fmt.Println("Error checking file:", err)
	}
}

func (self MissingImports) printLocalDate() {
	fmt.Println(time.Now().Format("2006-01-02"))
}

func (self MissingImports) printZoneId() {
	fmt.Println(time.Now().Location())
}
